cyclops-ui/src/components/pages/ModuleDetails.tsx

```diff
@@ -862,6 +897,16 @@ const ModuleDetails = () => {
         cancelButtonProps={{ style: { display: "none" } }}
         width={"40%"}
       >
+        <div>
+          <Divider>
+            <Checkbox
+            onChange={handleCheckboxChange} checked={showManagedFields} >
+              Include Managed Fields
+            </Checkbox>
+          </Divider>
+        </div>
+
+
```

Comment:
@Wasiim105 could you add the divider below the checkbox?
```suggestion
        <div>
          <Checkbox
            onChange={handleCheckboxChange} checked={showManagedFields} >
            Include Managed Fields
          </Checkbox>
          <Divider style={{marginTop: "12px", marginBottom: "12px"}}/>
        </div>
```